spec.html

```diff
@@ -35739,16 +36242,57 @@ <h1>Runtime Semantics: CompileToCharSet</h1>
           1. Assert: _p_ is a Unicode property name or property alias listed in the &ldquo;Property name and aliases&rdquo; column of <emu-xref href="#table-nonbinary-unicode-properties"></emu-xref>.
           1. Let _vs_ be SourceText of |UnicodePropertyValue|.
           1. Let _v_ be ! UnicodeMatchPropertyValue(_p_, _vs_).
-          1. Return the CharSet containing all Unicode code points whose character database definition includes the property _p_ with value _v_.
+          1. Let _A_ be the CharSet containing all Unicode code points whose character database definition includes the property _p_ with value _v_.
+          1. Return ! MaybeSimpleCaseFolding(_A_).
         </emu-alg>
         <emu-grammar>UnicodePropertyValueExpression :: LoneUnicodePropertyNameOrValue</emu-grammar>
         <emu-alg>
           1. Let _s_ be SourceText of |LoneUnicodePropertyNameOrValue|.
           1. If ! UnicodeMatchPropertyValue(`General_Category`, _s_) is identical to a List of Unicode code points that is the name of a Unicode general category or general category alias listed in the &ldquo;Property value and aliases&rdquo; column of <emu-xref href="#table-unicode-general-category-values"></emu-xref>, then
             1. Return the CharSet containing all Unicode code points whose character database definition includes the property &ldquo;General_Category&rdquo; with value _s_.
           1. Let _p_ be ! UnicodeMatchProperty(_s_).
-          1. Assert: _p_ is a binary Unicode property or binary property alias listed in the &ldquo;Property name and aliases&rdquo; column of <emu-xref href="#table-binary-unicode-properties"></emu-xref>.
-          1. Return the CharSet containing all Unicode code points whose character database definition includes the property _p_ with value &ldquo;True&rdquo;.
+          1. Assert: _p_ is a binary Unicode property or binary property alias listed in the &ldquo;Property name and aliases&rdquo; column of <emu-xref href="#table-binary-unicode-properties"></emu-xref>, or a binary Unicode property of strings listed in the &ldquo;Property name&rdquo; column of <emu-xref href="#table-binary-unicode-properties-of-strings"></emu-xref>.
+          1. Let _A_ be the CharSet containing all Unicode code points whose character database definition includes the property _p_ with value &ldquo;True&rdquo;.
```

Comment:
Maybe I'm missing something, but it looks to me like this still returns a set of code points, rather than ever returning a set of strings. Where are the actual semantics for `\p{Sequence_Property}`?

(It's possible I messed this up in my rebase, but looking at [the Google Doc](https://docs.google.com/document/d/1Tbv3hfX9CxQtzH9r-JdxJsQZhmmDsidRUKKxg345JV0/edit#heading=h.kijenta0fu5k) I don't see it there either.)

Reply:
It looks like my changes in the doc (“22.2.2.9.3 UnicodeMatchProperty ( p )” at the time) were incomplete. We need to rephrase this to say something like "p is a List of _CharSetElement_s...".

Reply:
Fixed: aba1390